models/common/fhir-path-reader.ts

```diff
@@ -0,0 +1,32 @@
+import {evaluate} from "fhirpath"
+import {Resource} from "fhir/r4"
+
+/*
+* Fhir Path Reader, used to extract values from FHIR.
+* see http://hl7.org/fhirpath/ for the specification
+* and https://github.com/HL7/fhirpath.js#readme
+* for the latest implementation
+*/
+
+export class FhirPathReader {
+    private fhir: Resource
```

Comment:
can we name this `fhirResource`? would be more readable when this is consumed on line 30

Reply:
Done ✔️